lib/UtapiClient.js

```diff
@@ -264,6 +275,40 @@ class UtapiClient {
     }
 
     /**
+    * Publish an event in the bucket's dedicated channel to notify subscribers
+    * for performed actions with side effects
+    * @param {string} metric - metric to be published
+    * @param {object} params - params for the metrics
+    * @param {string} [params.bucket] - (optional) bucket name
+    * @param {array} [params.objects] - (optional) objects affected
```

Comment:
Since `objects` can be confused with JS objects, can you call this param keys instead? The doc can also reflect what type of elements the array contains. For example, if it's an array of strings then
```
* @param {string[]} [params.keys] - (optional) objects in a bucket that are affected
```